sdk/iot/ci.yml

```diff
@@ -20,8 +20,9 @@ extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
     ServiceDirectory: iot
+    BuildSnippets: true
```

Comment:
If this is being done by default, do we need this here?   Maybe it was intended to be `false`?

Reply:
`true`, `false` - you say "toe-ma-toe" I say "toe-may-toe".

But, yes. :)